export/usb.py

```diff
@@ -53,18 +58,25 @@ def _process_usb(device):
 
 
 def main():
-    logging.basicConfig(level=logging.INFO)
+    logging.basicConfig(level=logging.DEBUG)
+
+    logging.debug(f'VIDEO_PATTERN:{VIDEO_PATTERN}')
+    logging.debug(f'MEDIA_DIR:{MEDIA_DIR}')
+    logging.debug(f'USB_MOUNT_DIR:{USB_MOUNT_DIR}')
+    logging.debug(f'USB_DIR:{USB_DIR}')
+
```

Comment:
You want to leave DEBUG statements in here?

Reply:
Yes I was planning to, it's helpful for debugging.  You can set the log level in the file to INFO if we want.